app.py

```diff
@@ -39,75 +52,109 @@ def daily():
 	except ValueError:
 		# Default to today (at midnight) if not
 		date = datetime.combine(datetime.today(), datetime.min.time())
-		
+	
 	# Formatted date strings when rendering page and buttons to other dates
 	date_str = date.strftime('%Y-%m-%d')
 	prev_date_str = (date-timedelta(1)).strftime('%Y-%m-%d')
 	next_date_str = (date+timedelta(1)).strftime('%Y-%m-%d')
 	
-	print("==== " + date_str + " ====" + "="*100)
+	tab_list = client_format.keys()
 	
-	# Determines what each tab/graph should display
-	with open(CLIENT_FORMAT_FILE) as file_handle:
-		client_format = json.load(file_handle)
-	
-	# Specifies information about each sensor in the database
-	with open(DATABASE_FORMAT_FILE) as file_handle:
-		db_format = json.load(file_handle)
+	try:
+		tab = request.args.get('tab')
+		if not tab in client_format.keys(): raise ValueError()
+	except ValueError:
+		tab = next(iter(client_format))
 	
+	#print("==== " + date_str: tab + " ====" + "="*100)	
 	
-	# Get the data from the database corresponding to the current date
-	# >= and <= are used instead of == due to time zone issues
-	db_data = db.collection(DATABASE_COLLECTION).where("date", ">=", date).where("date", "<=", date+timedelta(1)).stream()
-	try:
-		readings = next(db_data).to_dict()["telemetry"] # The map within the read document
-		readings = OrderedDict(sorted(readings.items())) # Sort the map
-		print("readings: " + str(readings))
-	except StopIteration:
-		readings = {} # No document for day requested
-	
-	
-	graph_data = OrderedDict() # The data rearranged for usage in the render template (see format below)
-	""" graph_data {
-			"Battery": {
-				"Voltage": {"1579970374000": 423, "1579970379000": 419, ...},
-				"Current": {"1579970374000": 46, "1579970379000": 48, ...},
-				...
-			}, "Solar": ...
+	graph_data = OrderedDict() # The data used in the render template (see format below)
+	""" graph_data = {
```

Comment:
Does this need to be left in the code? Not a big issue if it does, it's just not that great to have random comments in the code that's merged.

Reply:
The intention with this comment was to explain how graph_data was organized. It was more complicated when I added it, but now it's probably understandable enough without the example.